src/main/python/covata/delta/crypto/cryptoservice.py

```diff
@@ -12,155 +12,61 @@
-
-        :param str file_name: the name of the .pem file to be loaded
-        :return: the private key object
-        """
-        file_path = os.path.join(self.key_store_path, file_name)
-        with(open(file_path, 'r')) as f:
-            self.logger.debug("Loading %s", file_name)
-            return serialization \
-                .load_pem_private_key(f.read().encode('utf-8'),
-                                      password=self.__key_store_passphrase,
-                                      backend=default_backend())
-
-    @staticmethod
-    def generate_key():
-        # type: () -> rsa.RSAPrivateKey
-        """
-        Generates an RSA private key object. The public key object can be
-        extracted by calling public_key() method on the generated key object.
-
-        >>> private_key = CryptoService.generate_key() # generate a private key
-        >>> public_key = private_key.public_key() # get associated public key
-
-        :return: the generated private key object
-        """
-        return rsa.generate_private_key(public_exponent=65537,
-                                        key_size=4096,
-                                        backend=default_backend())
-
-    @staticmethod
-    def serialized(public_key):
-        # type: (rsa.RSAPublicKey) -> unicode
-        """
-        Serializes the provided public key object as base-64-encoded DER format
-        using X.509 SubjectPublicKeyInfo with PKCS1.
-
-        :param public_key: the public Key object
-        :type public_key: :class:`RSAPublicKey`
-        :return: the key as base64 encoded unicode string
-        :rtype: str
-        """
-        der = public_key.public_bytes(
-            encoding=serialization.Encoding.DER,
-            format=serialization.PublicFormat.SubjectPublicKeyInfo)
-        return base64.b64encode(der).decode(encoding='utf-8')
-
-    @staticmethod
-    def sha256hex(payload):
-        """
-        Calculates the SHA256 hex digest of the given payload.
-
-        :param str payload: the payload to be calculated
-        :return: SHA256 hex digest
-        :rtype: bytes
-        """
-        # type: (str) -> bytes
-        digest = hashes.Hash(hashes.SHA256(), backend=default_backend())
-        digest.update(payload if payload is bytes else payload.encode('utf-8'))
-        x = digest.finalize()  # type: bytes
-        return hexlify(x)
-
-    def signer(self, identity_id):
-        # type: (str) -> CVTSigner
-        """
-        Instantiates a new :class:`~.CVTSigner` for the authorizing identity
-        using this :class:`~.CryptoService`.
-
-        >>> signer = crypto_service.signer(authorizing_identity)
 
-        :param str identity_id: the authorizing identity id
-        :return: the CVTSigner object
-        :rtype: :class:`~covata.delta.crypto.CVTSigner`
-        """
-        return CVTSigner(self, identity_id)
+__all__ = ["generate_key", "serialize_public_key", "sha256hex"]
+
+
+def generate_key():
```

Comment:
Maybe call it `generate_private_key`? I mean I see we have `serialize_public_key` so looks like we explicitly trying to be clear. I don't know. If you feel like changing stuff :)